### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by PeteUgwu**
## React & Redux group project - Space Travelers' Hub

- Added Configurations & basic setups
- Redux: Fetched data and updated the Redux store 
- Rendered UI: lists
- Redux: Wrote actions and reducers for booking rockets/dragons and joining missions
- Redux: Wrote actions and reducers for canceling rockets/dragons and leaving missions
- Rendered UI: conditional components rendering
- Rendered UI: My Profile section
- Added test for all components and functions.

### CODE REVIEW

File: src/_tests_/Navbar.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+describe('Navbar tests', () => {
+  test('navbar render', () => {
+    const navbar = render(
+      <BrowserRouter>
+        <Navbar />
+      </BrowserRouter>,
+    );
+    expect(navbar).toMatchSnapshot();
+  });
+});
```
Comment (ghost) [inline src/_tests_/Navbar.test.js:15]: - [ ] Some of your tests are failing. Kindly fix that

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ Good Design


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/_tests_/Navbar.test.js (hunk 1 above)
Comment (umair-rao) [reply src/_tests_/Navbar.test.js:15]: Capture

Error in tests has been fixed, now all tests are passing. Please check.
Thank you.

**Review by EricMbouwe — APPROVED** [review]
Hi @umair-rao, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Test passing ✔
- PR ✔
- No linter errors ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**